### DISCUSSION TITLE
Slider "Jumps" when going to next slide

### DISCUSSION THREAD

**Discussion by chris087** [nolimits4web/swiper#4655] [Q&A] [answered]
We have a problem with the Swiper slider, please see:

http://eyee.fixio.nl/website-sociaal-planbureau-groningen/

The first slider, when going to the next slide, the whole slide with navigation "jumps".

Does anyone have an idea?

**Comment by nolimits4web** [comment]
Because you are animating active/inactive slides images sizes, so apparently it moves

**Reply by chris087** [reply]
it should not move.

**Comment by chris087** [comment, marked as answer]
giving .swiper-container a height fixed it.
